internal/store/project_ip_access_lists.go

```diff
@@ -22,25 +22,25 @@ import (
 	atlas "go.mongodb.org/atlas/mongodbatlas"
 )
 
-//go:generate mockgen -destination=../mocks/mock_project_ip_access_lists.go -package=mocks github.com/mongodb/mongocli/internal/store ProjectIPWhitelistDescriber,ProjectIPWhitelistLister,ProjectIPWhitelistCreator,ProjectIPWhitelistDeleter
+//go:generate mockgen -destination=../mocks/mock_project_ip_access_lists.go -package=mocks github.com/mongodb/mongocli/internal/store ProjectIPAccessListDescriber,ProjectIPAccessListLister,ProjectIPAccessListCreator,ProjectIPAccessListDeleter
 
-type ProjectIPWhitelistDescriber interface {
-	IPWhitelist(string, string) (*atlas.ProjectIPWhitelist, error)
+type ProjectIPAccessListDescriber interface {
+	IPAccessList(string, string) (*atlas.ProjectIPWhitelist, error)
 }
-type ProjectIPWhitelistLister interface {
-	ProjectIPWhitelists(string, *atlas.ListOptions) ([]atlas.ProjectIPWhitelist, error)
+type ProjectIPAccessListLister interface {
+	ProjectIPAccessList(string, *atlas.ListOptions) ([]atlas.ProjectIPWhitelist, error)
```

Comment:
super small nit, plural
```suggestion
	ProjectIPAccessLists(string, *atlas.ListOptions) ([]atlas.ProjectIPWhitelist, error)
```